Hi3Helper.Core/Lang/en.json

```diff
@@ -216,15 +216,15 @@
     "Graphics_APIPanel": "Graphics API",
     "Graphics_APIHelp1": "There are few API modes which are available for this game, including:",
     "Graphics_APIHelp2": "Note:",
-    "Graphics_APIHelp3": "Using DirectX 12 may cause crashes in some stages as the game wasn't natively compiled with DirectX 12 API",
+    "Graphics_APIHelp3": "Using DirectX 12 may cause crashes in some stages as the game wasn't natively compiled with the DirectX 12 API",
 
     "Graphics_SpecPanel": "Global Graphics Settings",
     "Graphics_Preset": "Graphics Preset",
     "Graphics_Render": "Rendering Accuracy",
     "Graphics_Shadow": "Shadow Quality",
     "Graphics_Reflection": "Reflection Quality",
     "Graphics_FX": "Visual Effects",
-    "Graphics_FXPost": "Post Proc.",
+    "Graphics_FXPost": "Post Processing",
```

Comment:
I think we need to leave it as is since it will overlap another text beside as shown in this screenshot:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/30566970/229307797-37202ba9-ccf4-45e3-8fc7-b8efdd8842c3.png">

Reply:
Can we adjust the padding to resolve the issue?

Reply:
How about just `Post Process.` ?